tool/build.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:build/build.dart';
```

Comment:
Any reason we're not using a build.yaml?

Reply:
+1 - this doesn't stand out to me as a case where we'd need a manual build script.

The import cycle against build_runner makes things tough, but I think not impossible.